woocommerce/woocommerce-gutenberg-products-block
# Implement new `Hero Product – Split` pattern

This adds a new pattern for a hero product banner with a 50/50 split, heading and CTA on left and product image on right.

Under the hood, it takes advantage of the Media + Image block, with some preset configurations.

![Hero-Split-Design](https://user-images.githubusercontent.com/481776/235234354-767fffaa-22ab-46ad-a882-60f277af27ae.png)



Fixes #9043 




### Screenshots



![CleanShot 2023-04-28 at 15 08 05@2x](https://user-images.githubusercontent.com/481776/235234292-4b8f7b6a-4846-439a-9266-5cea9e19fea4.png)

### Testing



1. Create a new page or post
2. Make sure the `WooCommerce Hero Product – Split` pattern appears under the WooCommerce category dropdown.
3. Insert in and make sure it shows as expected on the design.


### WooCommerce Visibility




### Performance Impact



### Changelog

> Add the new `WooCommerce Hero Product – Split` pattern.
